### DISCUSSION TITLE
neutral HDRI from the Model-viewer

### DISCUSSION THREAD

**Discussion by forestgunn** [google/model-viewer#4809] [Q&A]
Hi all. I am trying to find or make an HDRI that is used by the viewer in Neutral mode. Maybe there is a way to get this HDRI out of the viewer?  I want the  Blender viewport to match the viewer as closely as possible. any thoughts? There is a mirror sphere in ModelViewer with Neutral HDRI
![image](https://github.com/google/model-viewer/assets/136382652/35277119-ff36-4b98-a422-3cf0470c5f28)

**Comment by elalish** [comment]
Sure, it's here: https://github.com/google/model-viewer/blob/master/packages/shared-assets/environments/neutral.hdr

**Reply by snoyer** [reply]
Could you explain how this file was created? Was it generated and exported directly from the environment defined in `EnvironmentScene.ts`?
